lua/ii/ansible.lua

```diff
@@ -75,11 +75,13 @@ do return
   -- no setter for CMD 10 input
   }
 , getters =
-    { name = 'input'
-    , cmd  = 10 + get_offset
-    , args = { '', u8 }
-    --, docs = 'get input state'
-    , retval = { 'state', u8 }
-    }
+  { name = 'input'
+  , cmd  = 10 + get_offset
+  , args = { '', u8 }
+  --, docs = 'get input state'
+  , retval = { 'state', u8 }
+  }
+--void pickle( uint8_t* address, uint8_t* data, uint8_t* byte_count );
+, pickle = 'data[0] -= 1;  // zero-index the channel'
```

Comment:
I haven't tested on hardware but if I'm reading the code correctly I believe `data[0]` is possibly the command byte not the first argument since `ii_pickle` gets called after the `_ii_make_packet`

Reply:
i believe you are correct. will update & test on hardware (i don't have an ansible in my test case, but will set it up).